test/functional/p2p_invalid_block.py

```diff
@@ -82,6 +82,16 @@ def run_test(self):
 
         node.p2p.send_blocks_and_test([block2], node, success=False, request_block=False, reject_code=16, reject_reason=b'bad-txns-duplicate')
 
+        # Check transactions for duplicate inputs
```

Comment:
Why is 14247 partial? Why can't we do what's done in validation.cpp. If that was already done at some point, then this PR shouldN'T be marked partial